jujugui/static/gui/src/app/init/utils.js

```diff
@@ -76,8 +76,10 @@ utils.unloadWindow = function() {
   Export the YAML for the current model, including uncommitted changes.
 
   @param {Object} db The application database.
+  @param {Function} [sendAnalytics] Function to send analytics about the use of
+    the bundle export.
 */
-utils.exportEnvironmentFile = db => {
+utils.exportEnvironmentFile = (db, sendAnalytics)=> {
```

Comment:
This fn has no tests, and should have them. It uses so many globals that makes it quite difficult to properly test. Because we may remove this in favour of the Juju export I didn't feel it valuable to do that work at this time.